db.sql

```diff
@@ -2,4 +2,14 @@ CREATE TABLE utenti (
 	id int PRIMARY KEY
 );
 
-
+create table  caseificio (
```

Comment:
Le parole chiave devono essere in maiuscolo

Reply:
grazie